api/types/types_drivers_executor.go

```diff
@@ -131,7 +159,7 @@ type ProvidesStorageExecutorCLI interface {
 // StorageExecutorCLI provides a way to interact with the CLI tool built with
 // the driver implementations of the StorageExecutor interface.
 type StorageExecutorCLI interface {
-	StorageExecutorFunctions
+	StorageExecutorWithSupported
```

Comment:
Hurray, I get it!  Thanks a bunch for taking the time to explain that.

And, I didn't expect you to change anything. :)

However, I do like it better. :) Doesn't break anything as far as backwards compatibility goes, and now `StorageExecutorWithSupported` feels like a superset/enhancement.

Thanks again!